src/main/java/it/gov/pagopa/pu/classification/service/AssessmentsService.java

```diff
@@ -14,14 +20,41 @@
 public class AssessmentsService {
```

Comment:
Introduce an Interface!
Presentation, Service and Integration layer should communicate through Interface, not directly with implementation